src/macros.js

```diff
@@ -960,6 +960,15 @@ defineMacro("\\argmin", "\\DOTSB\\operatorname*{arg\\,min}");
 defineMacro("\\argmax", "\\DOTSB\\operatorname*{arg\\,max}");
 defineMacro("\\plim", "\\DOTSB\\mathop{\\operatorname{plim}}\\limits");
 
+//////////////////////////////////////////////////////////////////////
+// braket.sty
+// http://ctan.math.washington.edu/tex-archive/macros/latex/contrib/braket/braket.pdf
+
+defineMacro("\\bra", "\\mathinner{\\langle{#1}|}");
+defineMacro("\\ket", "\\mathinner{|{#1}\\rangle}");
+defineMacro("\\braket", "\\mathinner{\\langle{#1}\\rangle}");
+defineMacro("\\ketbra", "\\mathinner{|{#1}\\rangle\\langle{#2}|}");
```

Comment:
It seems `braket` doesn't define `\ketbra`:
> NOT defined is "\ketbra" (for projection operators) because I prefer \ket{ } \bra{ }.

Reply:
Hi, I've removed `\ketbra` and added `\Bra` and `\Ket` to be more faithful to the original `braket`.